Index: src/main/java/CastleWars/logic/CommandCentreRoom.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/CastleWars/logic/CommandCentreRoom.java b/src/main/java/CastleWars/logic/CommandCentreRoom.java
new file mode 100644
--- /dev/null	
+++ b/src/main/java/CastleWars/logic/CommandCentreRoom.java	
@@ -0,0 +1,10 @@
+package CastleWars.logic;
+
+import mindustry.content.Blocks;
+import mindustry.game.Team;
+
+public class CommandCentreRoom extends TurretRoom {
+    public CommandCentreRoom(Team team, int x, int y) {
+        super(team, Blocks.commandCenter, x, y, 1000, 4);
+    }
+}
